.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -13,7 +13,7 @@ A clear and concise description of what the bug is.
 **Application version**
 Version number: [e.g. 1.2.3]
 
-<!-- See application version under "Mitt AtB" screen -->
+<!-- See application version under "My profile" screen -->
```

Comment:
Hva er det egentlig snakk om her, @rosvik? Sånn jeg forstår den lille setninen her vil det kanskje være mer riktig å si "See application version at the bottom of your "My profile" page.", i stedet for at vi snakker om "My profile screen".

Reply:
@strombraaten Dette er teksten som dukker opp når du oppretter en bug report issue på github. ("New issue" -> "Bug report") Altså er det bare oss på teamet (evt. engasjerte brukere som skulle oppdage GitHub repoet) som ser dette.

Reply:
> @strombraaten Dette er teksten som dukker opp når du oppretter en bug report issue på github. ("New issue" -> "Bug report") Altså er det bare oss på teamet (evt. engasjerte brukere som skulle oppdage GitHub repoet) som ser dette.

Ah, da er det jo ikke så farlig nei.